### ISSUE TITLE
The updateNewMessage event is not triggered unless you enter the message in the Telegram application and see it.

### ISSUE THREAD

**Issue by AktilaCengiz** [tdlib/td#2924]
Hello everyone, 
The updateNewMessage event is not triggered unless you enter the message in the Telegram application and see it.
At the same time, there is another self bot on the machine and such an event does not occur. I wonder if it can't keep two connections open at the same time from a machine?

**Comment by levlam** [comment]
Check that you use new enough TDLib version and that you have a few devices in the active device list.

**Comment by AktilaCengiz** [comment]
I use TDLİB 1.8.9
![resim_2024-06-10_141708492](https://github.com/tdlib/td/assets/49373230/5dff4c99-fd02-4019-83b3-22d5d84d8d41)

Three different records appear to be open in the account

**Comment by levlam** [comment]
This is a very old version.

**Comment by predators46** [comment]
> This is a very old version.

same tdlib 1.8.30

**Comment by AktilaCengiz** [comment]
However, there was a problem like this: I did not receive any messages to the normal client. I saw that I received a message from a different account of mine. Then, when I refreshed it, it worked.

**Comment by benwoniu630353** [comment]
This problem does exist! And this problem has appeared in the past week, and it did not exist before!
The version number we use is 1.8.30, and the situation we encounter is the same as the problem encountered by the questioner above. When joining a new group/channel, there are only two situations to obtain the chatID:
01, log in again by receiving the verification code;
02, when there is a new message sent to the group/channel;

**Comment by levlam** [comment]
It is up to the server, when notify apps about new messages.

**Comment by benwoniu630353** [comment]
翻译
You didn't understand the problem we described.
Let me rephrase it:
Suppose there is a telegram account with a mobile phone number of 123. I log it in to the mobile telegram, and then also log in to the app developed based on tdlib 1.8.30.
I follow a channel on telegram on the mobile phone, and then use the app developed by tdlib to get all the chatlists with the current login account 123. At this time, the channel I just followed will not be displayed unless there are new messages sent from the channel!

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by levlam** [comment]
@benwoniu630353 
This is still up to the server.

Also, check that you have a few devices in the active device list.

**Comment by benwoniu630353** [comment]
4 devices

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AktilaCengiz** [comment]
3 device for me.

**Comment by esaaprillia** [comment]
This problem has been around for a long time

https://github.com/vysheng/tg/issues/1758

**Comment by AktilaCengiz** [comment]
This problem about large group or channel. Please use `openChat` then solve it.
